cordova-lib/src/PlatformProject.js

```diff
@@ -257,11 +257,15 @@ function save() {
 PlatformProject.prototype.build = build;
 function build(opts) {
     var self = checkThis(this);
-    var bin = path.join(self.root, 'cordova', 'build');
-    var args = [];
 
-    var copts = { stdio: 'inherit' };
-    return superspawn.spawn(bin, args, copts);
+    return self.copyWww(opts.paths.www)
```

Comment:
I guess I agree.  I just wanted to move it out of create for now, but you are right that it should be its own step.